dev/ci/ci-user-overlay.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Add user overlays here. You can use some logic to detect if you are
+# in your travis branch and conditionally enable the overlay.
```

Comment:
Sounds good, however there is the case when non-backward compatible changes are needed, like it is already the case in plugins due to interface changes. In this cases, you want to merge first the Coq PR, then the users.